packages/babel-core/src/config/config-chain.ts

```diff
@@ -779,6 +780,20 @@ function configFieldIsApplicable(
   return matchesPatterns(context, patterns, dirname);
 }
 
+/**
+ * Print the ignoreList-values in a more helpful way than the default.
+ */
+function ignoreListReplacer(
+  _key: string,
+  value: IgnoreList | IgnoreItem,
+): IgnoreList | IgnoreItem | string {
+  if (value instanceof RegExp) {
+    return inspect(value);
```

Comment:
I don't think we need to use `util.inspect`:

```suggestion
    return String(value);
```

Reply:
That's awesome! I'll fix it